daemon/inspect.go

```diff
@@ -59,7 +66,22 @@ func (daemon *Daemon) ContainerInspect(name string) (*types.ContainerJSON, error
 		StartedAt:  container.State.StartedAt,
 		FinishedAt: container.State.FinishedAt,
 	}
-
+	volumes := make(map[string]string)
+	volumesRW := make(map[string]bool)
+	for _, v := range container.volumes {
+		config := container.VolumeConfig[v.Name()]
+		path, err := v.Mount()
+		if err != nil {
+			return nil, err
+		}
+		volumes[config.Destination] = path
+		volumesRW[config.Destination] = config.RW
+		v.Unmount()
```

Comment:
We have to call Mount to get the actual path that is returned in the inspect output so we also need to call the unmount call to decrement the Mount call